### PR TITLE
chore: GitHub Actions CI

### PR DESCRIPTION
**Pull request by alexander-fenster**
Introducing GitHub Actions CI, pretty much the same setup as in [googleapis-discovery](https://github.com/googleapis/googleapis-discovery/blob/master/.github/workflows/ci.yaml).

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        echo -n "Cache found. Cache size: "
+        du -sh ~/.cache/bazel
+        echo "If the cache seems broken, update the CACHE_VERSION secret in"
+        echo "https://github.com/googleapis/disco-to-proto3-converter/settings/secrets/actions"
+        echo "(use any random string, any GUID will work)"
+        echo "and it will start over with a clean cache."
+        echo "The old one will disappear after 7 days."
+
+    - name: Run bazel build in google/cloud/compute/v1
+      run: bazel build '//google/cloud/compute/v1/...'
```
Comment (vam-google) [inline .github/workflows/ci.yaml:45]: Please add bazel test as well

File: .github/workflows/ci.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+    - name: Cache Bazel files
+      id: cache-bazel
+      uses: actions/cache@v2
+      with:
+        path: ~/.cache/bazel
+        key: ${{ runner.os }}-bazel-20210105-${{ secrets.CACHE_VERSION }}
+
+    - name: Cache not found
+      if: steps.cache-bazel.outputs.cache-hit != 'true'
```
Comment (vam-google) [inline .github/workflows/ci.yaml:29]: It is fine to have cache for consistency with googleapis-discovery, but strictly speaking it is not necessary here, this repo is very small.

File: .github/workflows/ci.yaml (hunk 1 above)
Comment (alexander-fenster) [reply .github/workflows/ci.yaml:45]: It fails because there are no tests (I tried!)

File: .github/workflows/ci.yaml (hunk 2 above)
Comment (alexander-fenster) [reply .github/workflows/ci.yaml:29]: Does not hurt I guess?

File: .github/workflows/ci.yaml (hunk 1 above)
Comment (alexander-fenster) [reply .github/workflows/ci.yaml:45]: The failed run (with test): https://github.com/googleapis/disco-to-proto3-converter/runs/1653748792

Comment (alexander-fenster) [reply .github/workflows/ci.yaml]: Changed to `//...` and added `bazel test`.

**Comment by alexander-fenster** [comment]
Successful test run: https://github.com/googleapis/disco-to-proto3-converter/pull/22/checks?check_run_id=1658154894

**Review by vam-google — APPROVED** [review]
LGTM

**Pull request merged**